### PR TITLE
First command in orb-utils to install the new CLI

### PR DESCRIPTION
**Pull request by ndintenfass**

### CODE REVIEW

File: src/orb-utils/orb.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2.1
+commands:
+  install-circleci-cli:
+    steps:
+      - run:
+          name: "Install `circleci` CLI"
+          command: |
+            sudo bash -c "$(curl -fSl https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh)"
+            echo "Run circleci help"
+            circleci help
```
Comment (bradylill) [inline src/orb-utils/orb.yml]: Is it worth running `circleci version` before help?

File: src/orb-utils/orb.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2.1
+commands:
+  install-circleci-cli:
+    steps:
+      - run:
+          name: "Install `circleci` CLI"
+          command: |
+            sudo bash -c "$(curl -fSl https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh)"
```
Comment (bradylill) [inline src/orb-utils/orb.yml]: Do we want to do some kind of a integrity check in that install script? (Not really part of this PR)

File: src/orb-utils/orb.yml (hunk 1 above)
Comment (ndintenfass) [reply src/orb-utils/orb.yml]: Now that we have easy orbs in orbs, this should probably be an orb called `circleci/cli` or maybe `circleci/circleci-cli`, so that it can be used for thing other than orb development. The original card also imagined there would be an image with it, so one could call that as an executor. As-is, this definitely assumes the presence of bash and curl, which is probably fine for now, but one reason to have the image was so that I could spin up a job to work with CircleCI trivially if I have the orb with the CLI.

Comment (zzak) [reply src/orb-utils/orb.yml]: :+1: for the CLI being it's own orb. I like `circleci/circleci-cli`.

File: src/orb-utils/orb.yml (hunk 2 above)
Comment (zzak) [reply src/orb-utils/orb.yml]: Like using a checksum?

File: src/orb-utils/orb.yml (hunk 1 above)
Comment (zzak) [reply src/orb-utils/orb.yml]: @ndintenfass Added the executor, which depends on this PR getting merged:
https://github.com/CircleCI-Public/circleci-cli/pull/78

File: src/orb-utils/orb.yml (hunk 2 above)
Comment (bradylill) [reply src/orb-utils/orb.yml]: sha256 would work well

**Comment by ndintenfass** [comment]
LGTM
